echopype/convert/api.py

```diff
@@ -294,23 +308,48 @@ def _check_file(
     else:
         xml = ""
 
+    # Check .bot file
+    if SONAR_MODELS[sonar_model]["accepts_bot"] and include_bot:
+        bot_file = str(Path(raw_file).with_suffix(".bot"))
+        bot_fsmap = fsspec.get_mapper(bot_file, **storage_options)
+        if not bot_fsmap.fs.exists(bot_fsmap.root):
+            raise FileNotFoundError(
+                f"There is no file named {bot_file}. The `.BOT` file must be contained in "
+                + " the same directory as that of the input 'raw' file."
```

Comment:
This is super tiny but i think since the backtick is really used for docstring, maybe we can skip it here.
```suggestion
                f"There is no file named {bot_file}. The .BOT file must be contained in "
                + " the same directory as that of the input 'raw' file."
```